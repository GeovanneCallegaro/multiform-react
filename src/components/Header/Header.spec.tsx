import { render, screen } from "@testing-library/react"
import { Header } from "."

export {}

describe('<Header />', () => {
  it('should have a title and a paragraph on the screen', () => {
    render(<Header />);

    expect(screen.getByRole('heading', {name: /Cadastro de Desenvolvedor/i})).toBeInTheDocument();
    expect(screen.getByText(/faÃ§a seu cadastro na lista/i)).toBeInTheDocument();
  })

  it('should match snapshot', () => {
    const {container} = render(<Header />)

    expect(container.firstElementChild).toMatchSnapshot()
  })
})