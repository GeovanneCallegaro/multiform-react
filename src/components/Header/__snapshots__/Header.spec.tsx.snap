// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match snapshot 1`] = `
<div
  class="sc-bczRLJ OYxqk"
>
  <h1>
    Cadastro de Desenvolvedor
  </h1>
  <p>
    Fa√ßa seu cadastro na lista e receba um emprego instantaneamente
  </p>
</div>
`;
